app/assets/scss/_forms.scss

```diff
@@ -194,11 +194,19 @@ main {
     }
     /* hide optional */
     #sellerEmailList .question-optional {
-      display: none
+      display: none;
     }
     /* hide optional */
     #sellerEmail .question-optional {
-      display: none
+      display: none;
+    }
+
+    #trainingDetailType .question-optional {
+      display: none;
+    }
+
+    #trainingDetailCover .question-optional {
+      display: none;
     }
```

Comment:
Can you use a single selector here?  Like: `.question-optional { display: none; }`

Reply:
no, because this would hide for all optional questions